tests/staticfiles_tests/test_checks.py

```diff
@@ -132,3 +132,54 @@ def test_nonexistent_directories(self):
             # Nonexistent directories are skipped.
             finder = get_finder("django.contrib.staticfiles.finders.FileSystemFinder")
             self.assertEqual(list(finder.list(None)), [])
+
+
+class StorageCheckTests(SimpleTestCase):
+    DEFAULT_STORAGE_VALUE = {
+        "default": {
+            "BACKEND": "django.core.files.storage.FileSystemStorage",
+        },
+        "staticfiles": {
+            "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+        },
+    }
+
+    @staticmethod
+    def _remove_storage_alias(alias):
+        """
+        Helper function to remove an alias from the STORAGES setting.
+
+        We need this method because UserSettingsHolder has some backwards compatible
+        code in __setattr__ to keep the aliases 'default' and 'staticfiles' which
+        prevents us from testing what happens when they are missing.
+        """
+        settings.STORAGES.pop(alias, None)
+
+    def test_no_errors_without_setting(self):
```

Comment:
If the goal of this test is to ensure that, when `STORAGES` is not defined, no error is raised, I think this test should explicitely unset `STORAGES`? What do you think?

Reply:
I assumed that if I don't touch it, it would use [this set of settings](https://github.com/django/django/blob/9c19aff7c7561e3a82978a272ecdaad40dda5c00/tests/staticfiles_tests/settings.py), where `STORAGES` it's not defined, and that was enough. How do you suggest unsetting it exactly?

Reply:
The thing is that the final value for `STORAGES` is the whole dictionary as [described in the docs](https://docs.djangoproject.com/en/4.2/ref/settings/#storages), so I went ahead and explicitly cleared the dict.